### ISSUE TITLE
Feature request: config option to disable overriding vanilla ore textures

### ISSUE THREAD

**Issue by nullifiedcat** [MinecraftModDevelopmentMods/MinecraftMineralogy#64]
The new ore textures look out of place to me, I'd like Vanilla textures back.

**Comment by carstorm** [comment]
Agreed,perhaps it could be set to a  client side config option?

**Comment by SkyBlade1978** [comment]
that would be nice, well its on the to-do list now, either I'll have a go at it sometime unless some kind soul does it in a PR..

**Comment by SkyBlade1978** [comment]
BTW as cyano said in his original blurb on the curseforge page it is possible to sort it out yourself by getting the vanilla textures and modifying the resource pack, but an easier solution would be nice

**Comment by carstorm** [comment]
> ...unless some kind soul does it in a PR..

I would if I knew how.

> getting the vanilla textures and modifying the resource pack...

IMO any mod that changes a vanilla texture sure have that feature as a client side config option

**Comment by SkyBlade1978** [comment]
I'll certainly do it, the todo list is growing faster than I can defeat it at the moment but it's near the top of the list I think, I'm also a big fan of client side config which is why the first PR I did to mineralogy was to add a bunch of config options :)

**Comment by Graywaren** [comment]
Agreed. This really should be a thing already. I want to disable the sandstone textures as I really dislike the dark details personally, and I don't want to have to go figure out how to modify the whole thing just to do that. 

Meanwhile... anyone have a link to a tutorial or info on how to do the whole resource pack modification thing?

**Comment by Graywaren** [comment]
Nvm about the tutorial. I figured it out. Here's the directions if anyone else wants to try.

If anyone else is looking to replace some of this mod's textures with Vanilla again here's how to do it:

First find the vanilla textures you want... I cobbled directions together from stack exchange posts:
On a windows PC, hold 'Win+R' and type %appdata% 
This should automatically go to Roaming, but if not simply go to the roaming folder. 
Navigate to the .minecraft directory
Go to the 'Versions' Folder. 
Find the version of minecraft you want the textures from, go to that folder. 
Open the .jar file for that version with an archive program like 7zip
Got to assests folder
minecraft folder
textures folder
blocks folder
copy out the textures that you want

Next for replacing the files in Mineralogy:
open the .jar file with the archive program
assets folder
minecraft folder
textures folder
blocks folder
verify the names of the .pngs you want to replace and make any edits to the vanilla files you saved out so they match
drag & drop those files into the archive window
double check they properly replaced then close the program

Tada.... you now should have the textures you want back.
